otelcol.exporter.otlphttp: new component (#2429)
grafana/agent: docs/sources/shared/flow/reference/components/otelcol-tls-config-block.md

---
aliases:
- /docs/agent/shared/flow/reference/components/otelcol-tls-config-block/
headless: true
---

The following arguments are supported:

Name | Type | Description | Default | Required
---- | ---- | ----------- | ------- | --------
`ca_file` | `string` | Path to the CA file. | | no
`cert_file` | `string` | Path to the TLS certificate. | | no
`key_file` | `string` | Path to the TLS certificate key. | | no
`min_version` | `string` | Minimum acceptable TLS version for connections. | `"TLS 1.2"` | no
`max_version` | `string` | Maximum acceptable TLS version for connections. | `"TLS 1.3"` | no

> `insecure` | `boolean` | Disables TLS when connecting to a gRPC or HTTP server. | | no

Reviewer: Ditto for dropping the qualifier here:

```suggestion
`insecure` | `boolean` | Disables TLS when connecting to the configured server. | | no
```
